### ISSUE TITLE
Cant use an empty default for FlagProvider

### ISSUE THREAD

**Issue by chabad360** [BoRuDar/configuration#13]
It would be preferable if the `FlagProvider` respected the `default` tag, because otherwise it just introduces duplication, and if it's used before any other providers, all the other ones get ignored. The issue with not using a default is that its not possible to provide a description, because the second part of the tag is used for the default. If the default is left empty it's still used, meaning that all the other tags are still skipped.

Basically, `FlagProvider` should check if the `default` tag is present, if it is, then `FlagProvider` should not set a default (but should describe it as such in the usage output).

**Comment by BoRuDar** [comment]
Hi @chabad360,

Thank You for reporting. Mentioned bug is fixed now.

Best regards,
Bogdan
